### ISSUE TITLE
ARFoundations causes WebRequests to get stuck on Android

### ISSUE THREAD

**Issue by Sterling-Malory-Archer** [Unity-Technologies/arfoundation-samples#792]
I just experienced this while testing my app on Android, it was something I encountered on iOS but fixed it by looking at the answer from [this](https://github.com/Unity-Technologies/arfoundation-samples/issues/694#issuecomment-760437908).

My Unity version is 2019.4.0f1
My ARCore version is 4.1.0 - Preview 7
My AR Subsystems version is 4.1.0 - Preview 12
My ARFoundation version is 4.1.0 - Preview 7

Basically, what happens is when I detect an image, the WebRequest doesn't want to pull a new question, and instead just keeps loading the same question as before.

As I said I fixed it on iOS by adding the webOperationQueue.qualityOfService = NSQualityOfServiceUserInteractive; to the XCode project.

Any help is kindly appreciated!

**Comment by tdmowrer** [comment]
> Basically, what happens is when I detect an image, the WebRequest doesn't want to pull a new question, and instead just keeps loading the same question as before.

What do you mean by "pull a new question"?

Also, I recommend using the verified versions of AR Foundation instead of preview versions. The latest is 4.1.5.

**Comment by Sterling-Malory-Archer** [comment]
> > Basically, what happens is when I detect an image, the WebRequest doesn't want to pull a new question, and instead just keeps loading the same question as before.
> 
> What do you mean by "pull a new question"?
> 
> Also, I recommend using the verified versions of AR Foundation instead of preview versions. The latest is 4.1.5.

What I mean is when the user recognizes the correct "image", they can then receive another "clue" about where the next one is. Currently, sometimes it gets "stuck" as in it only calls one WebRequest and it doesn't want to call another one.

Would downgrading from my current version affect anything regarding image recognition?

**Comment by Sterling-Malory-Archer** [comment]
Hi Tim,

I tried upgrading to 4.1.5, but I got this error

Library\PackageCache\com.unity.xr.management@4.0.1\Editor\XRGeneralBuildProcessor.cs(36,52): error CS0117: 'BuildPipeline' does not contain a definition for 'GetBuildTargetName'

What can I do? I tried installing ScriptableBuildPipeline but that didn't fix it

**Comment by tdmowrer** [comment]
Update to the latest 2019.4 LTS.

**Comment by Sterling-Malory-Archer** [comment]
> Update to the latest 2019.4 LTS.

Ok, thank you, for now, I will try to update to 4.1.1 and see if that maybe fixes it.
